### ISSUE TITLE
Add automatic tags

### ISSUE THREAD

**Issue by everzet** [vipsoft/JiraExtension#2]
You should transform useful jira information:
1. Assigned users
2. Assigned milestone
3. Assigned tags

into feature tags, as GithubExtension does:

``` gherkin
@assignee:everzet @milestone:sprint_1 @bug
Feature: ...
  ...
```

This way, you'll be able to use internal Behat filtering abilities to run only user/milestone/label features.

**Comment by robocoder** [comment]
Ok, I'll check the API.  I think this is doable.

**Comment by robocoder** [comment]
Partially implemented in v1.0.2

Jira has the concept of "affected versions" and "fix versions".  The latter is generally the targeted-fix-for-version, and is probably the closest conceptually to milestone.

Jira doesn't have tags. There is a 3rd party plugin and the core ability to use custom fields.  Neither is really usable via the SOAP API, so deferring until issue #3 (Jira 5 REST API).
